Packages/com.unity.inputsystem/InputSystem/Controls/ButtonControl.cs

```diff
@@ -56,25 +59,38 @@ public class ButtonControl : AxisControl
         /// </code>
         /// </example>
         /// </remarks>
-        /// <seealso cref="InputSettings.defaultButtonPressPoint"/>
```

Comment:
Rather than just removing these you can embed into the remarks section using the following in a sentence.
It is worth doing this as the links are useful.

`<see cref="InputSettings.defaultButtonPressPoint"/>`